src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/LibraryImportAttribute.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+namespace System.Runtime.InteropServices
+{
+    /// <summary>
+    /// Attribute used to indicate a Source Generator should create a function for marshaling
+    /// arguments instead of relying on the CLR to generate an IL Stub at runtime.
+    /// </summary>
```

Comment:
We should consider adding remarks about how the attribute is meaningless if the generator isn't being used, and the generator is currently only available for code written in C#.

Reply:
```suggestion
    /// </summary>
    /// <remarks>
    /// This attribute is meaningless if the source generator associated with it is not enabled.
    /// The associated source generator only supports C# and only supplies an implementation when
    /// applied to static, partial, non-generic methods.
    /// </remarks>
```